### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by maypfv**

### CODE REVIEW

**Review by asdfghjkxd — APPROVED** [review]
LGTM

File: docs/team/seantehds.md [hunk 1]
```diff
@@ -8,40 +8,41 @@ title: Teh De Sheng Sean's Project Portfolio Page
...
-        * Therefore, I updated the Add Command from 1.2 to be able to take in arguments, and create a task with these fields already initialised to these given inputs.
+### Enhancement To Existing Features
+* Updated the Add Command to allow for variable arguments from the user: [#127](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/127)
+  * Before 1.3, the Add Command was only able to take in a description input from the user in order to create a task within TaskWise. This would result in a task being created with all other fields, like deadlines and priorities being initialised to their default values.
+  * As a group, we believed that the user experience would be enhanced if users were given an option to create a task with certain specified fields already initialised to values already given by the user's input.
+  * Therefore, I updated the Add Command from 1.2 to be able to take in arguments, and create a task with these fields already initialised to these given inputs.
```
Comment (seantehds) [inline docs/team/seantehds.md:42]: Hi May, could you change this line to be this instead? 

Therefore, I updated the Add Command from 1.2 to be able to take in additional arguments that were optional, and create a task with these fields already initialised to these given inputs.

File: docs/team/seantehds.md [hunk 2]
```diff
@@ -8,40 +8,41 @@ title: Teh De Sheng Sean's Project Portfolio Page
...
     * Credits: Sebastian for implementing the Status class and field necessary for tracking whether the
       task has been completed or not.
-* **New Feature**: Unmark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
+* **New Feature:** Unmark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
     * What it does: Marks a task within the task list as incomplete.
     * Justifications: Allows the user to mark tasks as incomplete to allow the task manager to keep track of
       tasks and whether they are finished or not.
     * Highlights: The task becomes opaque when it is marked as incomplete.
     * Credits: Sebastian for implementing the Status class and field necessary for tracking whether the
       task has been completed or not.
```
Comment (seantehds) [inline docs/team/seantehds.md:30]: Could you help me change this line over to this particular line:
Credits: Sebastian for implementing the Status class and field necessary for tracking whether the task has been completed or not, as well as the change in opacity of the task card depending on status.

File: docs/team/seantehds.md [hunk 3]
```diff
@@ -8,40 +8,41 @@ title: Teh De Sheng Sean's Project Portfolio Page
...
 
 ### New Features and Components
 
-* **New Feature**: Mark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
+* **New Feature:** Mark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
     * What it does: Marks a task within the task list as completed.
     * Justifications: Allows the user to mark tasks as completed to allow the task manager to keep track of
       tasks and whether they are finished or not.
     * Highlights: The task reduces in opacity when it is marked as completed.
     * Credits: Sebastian for implementing the Status class and field necessary for tracking whether the
```
Comment (seantehds) [inline docs/team/seantehds.md:22]: Could you help me change this line over to this particular line:
Credits: Sebastian for implementing the Status class and field necessary for tracking whether the task has been completed or not, as well as the change in opacity of the task card depending on status.

**Review by JCSnap — APPROVED** [review]

**Pull request merged**